2022-08-07-qubes-os-summit-history.md

```diff
@@ -0,0 +1,97 @@
+given by Qubes OS and 3mdeb experts on
+[Youtube](https://www.youtube.com/playlist?list=PLuISieMwVBpIwhPXcuYKtS50CHQOvt_BO)
+over and over again. It was quite a challenge for us because we have organized
+such a virtual event for the first time. Fortunately, everything went smoothly
+and finished with success after 4 sessions 2.5 hours long.
+
+In August 2021 we held yet another Qubes OS Summit together with the Qubes OS
+team, still virtually, unfortunately. However, this year brought more speakers
+from other companies and projects, which we were very happy about. There were 2
+sessions almost 4 hours each and one may watch the talks at
+[Youtube](https://www.youtube.com/playlist?list=PLuISieMwVBpIoLQzpYeZnkupURheXky6r).
+
+# The future of Qubes OSe Summits
+
+Finally, the time has come when we may meet together in person, after two
+years! The Qubes OS Summit 2022 is fast approaching, with only a few days left
+before the start of the event in Berlin. As it is no longer held only virtually
+(but still possible to attend remotely for those who can be with us in person)
+the organization is more complicated, both from the logistics and presentations
+side. Every year we try our best to dig into hardcore security topics on the
+verge of firmware and operating systems. All this time we have been gradually
+building plans and ideas, and getting closer and closer to the goal of
+improving the security of Qubes OS, especially in areas we feel best at, i.e.
+SRTM and DRTM (AEM). In the upcoming event, we have prepared a few interesting
+projects and ideas around Anti Evil Maid support and open-source firmware on
+modern hardware. [Here](https://cfp.3mdeb.com/qubes-os-summit-2022/schedule/#)
+you may find the full schedule. Among others we welcome the following speakers:
+
+* Wessel Klein Snakenborg from Novacustom with a story of open-source on modern
+  laptops as a candidate to Qubes OS certification
+* Brent Cowing from Protectli talking about security of his small form factor
+  computers also running open-source firmware
+* Jan Suhr from Nitrokey talking about how enterprise requirements tied to
+  Windows can be met with Qubes OS
+* Arthur Rasmusson from OpenMdev project about GPU virtualization and its
+  possible benefits to QUbes oS
+* Puck Meerburg from Spectrum OS telling about the power of Wayland for GUI
+  isolation and how it benefits systems like Qubes OS or Spectrum OS
+* And and everyone from 3mdeb, Qubes OS team and others
+
+We hope that the event and community around it and the Qubes OS project will
+grow in the future, bringing benefits and collaboration for everybody.
+Definitely, 3mdeb will try its best to co-organize the event in the upcoming
+years.
+
+# Summary
+
+By a mere coincidence, meeting Marek Marczykowski-Górecki gave birth to a
+fascinating series of Qubes OS Summits and led us to this day.
+
+We (3mdeb) are very proud to be the co-organizers of the 4th edition of the
```

Comment:
Maybe rephrase that in way as follows:

Together with InvisibleThingsLab we (3mdeb) are very proud ....

Reply:
Done